client/views/windows/settings.tpl

```diff
@@ -5,6 +5,19 @@
 	<h1 class="title">Settings</h1>
 
 	<div class="row">
+		{{#unless public}}
+		<div class="col-sm-12">
+			<h2>
+				Settings synchronisation
+			</h2>
+			<label class="opt">
+				<input type="checkbox" name="syncSettings">
+				Synchronize settings with other clients.
+			</label>
+			<span class="sync-warning-override"><strong>Warning</strong> Checking this box will override the settings of this client with those stored on the server.</span>
+			<span class="sync-warning-base"><strong>Warning</strong> No settings have been synced before. Enabling this will sync all settings of this client as the base for other clients.</span>
```

Comment:
Could use `p` elements instead of `span`s? That will fix the extra font size there is right now.

Reply:
I actually liked the warning being slightly bigger, but yeah it can also be changed if you prefer so.

Reply:
That would be great, thanks :)